web/components/auth/Login/Login.js

```diff
@@ -1,140 +1,69 @@
-      this.setState({
-        error: {},
-      });
-    }).catch((error) => {
-      this.setState({ error });
-    });
-  }
-
-  loginView() {
-    const { email, password, error } = this.state;
-    return (
-      <div id="login" className="fill-height flex jc-center ai-center dir-col">
-        <div>
-          <form
-            onSubmit={this.login}
-            className="input-group"
-            name="login-form">
-            <h3 className="pad-sides-xxl margin-bottom-m">Sign in</h3>
-            <div className="card split flex">
-              <div className="pad-ends-xxl pad-sides-xxl margin-vertical-text-inputs">
-                <TextInput
-                  placeholder="Enter your email"
-                  value={email}
-                  onChange={this.onEmailChange}
-                  error={error}
-                  label="Email"
-                  name="email"
-                />
-                <PasswordInput
-                  placeholder="Enter your password"
-                  value={password}
-                  onChange={this.onPasswordChange}
-                  error={error}
-                  label="Password"
-                  name="password"
-                  showErrorMessage
-                  showForgot
-                />
-                <PrimaryButton
-                  text="Submit"
-                  type="submit"
-                  className="fill-width"
-                />
-                <p className="flex jc-center">
-                  Don&apos;t have an account yet?&nbsp;
-                  <Link to="/">Apply here</Link>
-                </p>
-              </div>
-              <div className="card-right" />
-            </div>
-          </form>
+const Login = ({
+  onEmailChange,
+  onPasswordChange,
+  login,
+  email,
+  password,
+  error,
+}) => (
+  <div id="login" className="fill-height flex jc-center ai-center dir-col">
+    <div>
+      <form
+        onSubmit={login}
+        className="input-group"
+        name="login-form">
+        <h3 className="pad-sides-xxl margin-bottom-m">Sign in</h3>
+        <div className="card split flex">
+          <div className="pad-ends-xxl pad-sides-xxl margin-vertical-text-inputs">
+            <TextInput
+              placeholder="Enter your email"
+              value={email}
+              onChange={onEmailChange}
+              error={error}
+              label="Email"
+              name="email"
+              />
+            <PasswordInput
+              placeholder="Enter your password"
+              value={password}
+              onChange={onPasswordChange}
+              error={error}
+              label="Password"
+              name="password"
+              showErrorMessage
+              showForgot
+              />
+            <PrimaryButton
+              text="Submit"
+              type="submit"
+              className="fill-width"
+              />
+            <p className="flex jc-center">
```

Comment:
Can we comment this out for now to avoid any confusion?
https://github.com/nwplus/nwhacks2019/blob/2bf35852b128dcdd2d95f60d0d970f79ad3803e7/web/components/auth/Login/Login.js#L48-L51